src/styles/booleanDropdown.scss

```diff
@@ -0,0 +1,52 @@
+@import 'global';
+
+.drop {
+  background-color: white;
+  border-radius: 3px;
+  font-family: $main-font;
+  font-size: 20px;
+  line-height: 23px;
+  margin-bottom: 3px;
+  min-height: 27px;
+  padding-left: 10px;
+  position: absolute;
+  width: 60px;
+}
+
+.top {
+  top: 0;
+  z-index: 3;
+}
+
+.middle {
+  top: 30px;
+  z-index: 2;
+}
+
+.bottom {
+  top: 60px;
+  z-index: 1;
+}
+
+.is-open {
+  background-color: lightgray;
+}
+
+.arrow {
+  background: url('../assets/arrow.png') no-repeat right;
```

Comment:
try to prefer svgs! they are more scalable + less data. In this case, it's not a huge deal, but still good to keep it consistent 

Here's how you can do it:
![image](https://user-images.githubusercontent.com/29292203/123525607-df955680-d686-11eb-9339-79a87d0dd5f4.png)

notice there isn't an overload of data:
```
<svg width="10" height="8" viewBox="0 0 10 8" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M5 7.64285L0.669872 0.678567L9.33013 0.678567L5 7.64285Z" fill="#FF8A00"/>
</svg>
```